lib/dockerregistry/inventory_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Hmm, you don't have to update this. Instead you should update the year for your `checks.go`.

Reply:
yeah, this was a mistake. I was intending to update the year for the checks_test.go